oresat_dxwifi/resources/temperature.py

```diff
@@ -0,0 +1,119 @@
+# Purpose:
+#       Issue: https://github.com/oresat/oresat-dxwifi-software/issues/1.
+#       The temperature "read" by an NTC thermistor is calculated based on the voltage recieved at the ADC input pin.
+#       The calculations are based on https://docs.google.com/document/d/16b_Qd6Pj2IAk5HQQl7JAP1T9Qd6QbKICMd-0swd9L6Y/edit
+#       The resource then sends the calculated temperature to the Object Directory upon resource start. An SDO read callback is also implemented
+
+from olaf.common.resource import Resource
+
+from numpy import log as ln
+from olaf.common.adc import Adc
+
+class TemperatureResource(Resource):
+    """Resource for getting the temperature of the NTC thermistor connected at AIN6"""
+
+    # Olaf Constants ------------------------------------------------------------------------------
+
+    TEMPERATURE_INDEX = 0x6001
+    MOCK_ADC = False # False = real world voltage, True = fake voltage, this constant is for testing
+
+    # Resource Related ----------------------------------------------------------------------------
+
+    def __init__(self):
+        super().__init__()
+        self.index = self.TEMPERATURE_INDEX
+        self.ADC = Adc(self.ADC_THERMISTOR_PIN, self.MOCK_ADC)
```

Comment:
change to `self.adc`.  All caps variables should be reserved for constants.

Reply:
Done.